rule/add_constant.go

```diff
@@ -212,16 +212,16 @@ func (r *AddConstantRule) Configure(arguments lint.Arguments) error {
 	}
 	for k, v := range args {
 		kind := ""
-		switch k {
-		case "allowFloats":
+		switch ruleOptionFromArg(k) {
+		case "allowfloats":
```

Comment:
I wouldn't change existing variable case here, it doesn't help with readability.

What about renaming `ruleOptionFromArg` to `normalizeRuleOption`


```suggestion
		switch normalizeRuleOption(k) {
		case normalizeRuleOption("allowFloats"):
```

It comes with extra computing, but it could help to keep camelCase readability